BUILD.md

```diff
@@ -1,163 +1,162 @@
 Build and publish workflow
 ===========================
 
-Main project target is to release the artifacts to one of central dependencies repository 
-(Maven central so far, but might be Bintray in the future). Unfortunately there is no so far any consistent way to 
-publish the project using TravisCI, so manual publishing is required.
-
-Table of content:
-
-  - [Integration tests](#integration-tests)
-  - [Publish workflow](#publish-workflow)
-    - [Publish to local maven repo](#publish-to-local-maven-repo)
-    - [Publish to Maven](#publish-to-maven)
-      - [Signing up the app with PGP key](#2-signing-up-the-app-with-pgp-key)
-      - [Deploy to OSS Sonatype](#3-deploy-to-oss-sonatype)
-      - [Manually release from Sonatype web page to Maven Central](#4-manually-release-from-sonatype-web-page-to-maven-central)
-      - [Publish to repo.ci.cloud.commercetools.de](#5-publish-to-repocicloudcommercetoolsde)
-  - [All in one publish script](#all-in-one-publish-script)
-  - [Known issues](#known-issues)
-  
+The main goal of the build process is to publish the artifacts to public repositories, 
+like [JCenter](https://jcenter.bintray.com/) and [Maven Central](https://search.maven.org/).
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Build and deploy a new version](#build-and-deploy-a-new-version)
+- [Integration tests](#integration-tests)
+- [Publish workflow](#publish-workflow)
+  - [Full build with tests, documentation publishing and Bintray upload](#full-build-with-tests-documentation-publishing-and-bintray-upload)
+  - [Publish to local maven repo](#publish-to-local-maven-repo)
+  - [Publish to Bintray](#publish-to-bintray)
+  - [Publish to Maven](#publish-to-maven)
+- [All in one publish script](#all-in-one-publish-script)
+- [Known issues](#known-issues)
+
```

Comment:
don't think so: from my perspective build and publish are part of the same process, and they are tightly connected (publish is a part of build)